feat(ingest/tableau): enable extract_lineage_from_unsupported_custom_sql_queries by default

ligfx opened the pull request:
It only takes effect when Tableau itself returns no lineage, so it's strictly better than having it disabled.

sgomezvillamor reviewed (commented):
Even if this is an experimental feature, this should be announced as an breaking change in the `updating-datahub.md` docs.

Also we may take this as the opportunity to improve the description of these flags: what does "doesn't return any metadata" or "ignoring Tableau metadata" mean? do we refer to some specific API?

ligfx commented:
@sgomezvillamor I added a line to `updating-dathub.md` and reworked the description of the flags. Can you take another look when you get a chance and let me know what you think?
